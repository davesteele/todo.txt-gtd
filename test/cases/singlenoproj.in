This is a single todo item @nowhere
