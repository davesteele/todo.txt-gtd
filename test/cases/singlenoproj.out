# _None
#

This is a single todo item @nowhere
